### PR TITLE
Fixed a bug in prime_number.py

### PR DESCRIPTION
**Pull request by charlie219**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES

### Description

I noticed an issue with prime_number.py regarding the recursive approach. The condition to check (num==2) must be before then it is actually placed. which resulted in the wrong execution of the program.
For an input of 2, the recursive approach returned `False` which instead should be `True`.

I have also changed my profile link in `CONTRIBUTORS.md` file.

### Checklist

- [Y ] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

None

### CODE REVIEW

File: Python/math/prime_number.py [hunk 1]
```diff
@@ -40,10 +37,12 @@ def is_prime_iterative(num):
 def is_prime_recursive(num, i=2):
 
     # Base Cases
-    if(num <= 1 or n % i == 0):
+    if num == 2:
+        return True
```
Comment (atarax665) [inline Python/math/prime_number.py:41]: Add a line break after this.

**Review by atarax665 — APPROVED** [review]

File: Python/math/prime_number.py (hunk 1 above)
Comment (charlie219) [reply Python/math/prime_number.py:41]: Line Break Added

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@charlie219 Do not edit `CONTRIBUTORS.md` file. Contributors are added by the bot when the PR gets merged.

**Comment by charlie219** [comment]
@iamrajiv actually I am already added in `CONTRIBUTOR.md` for one of my merged PR, here I have just updated a link of my profile

**Review by iamrajiv — APPROVED** [review]

**Pull request merged**